deploy/docker/docker-compose.yml

```diff
@@ -65,5 +66,63 @@ services:
       - ogree_api
     restart: on-failure:10
 
+
+
+  arango_db:
+    image: arangodb/arangodb:3.11.2
+    container_name: ${COMPOSE_PROJECT_NAME}_arango_db
+    environment:
+        - ARANGO_ROOT_PASSWORD=${ARANGO_PASS}
+    volumes:
+      - arangodb-persist:/var/lib/arangodb3
+    ports:
+      - ${ARANGO_PORT}:8529
```

Comment:
this exposes the port of the database, doesn't it?

Reply:
Removed